presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -478,6 +480,19 @@ public void testSTEnvelope()
         assertFunction("ST_AsText(ST_Envelope(ST_GeometryFromText('GEOMETRYCOLLECTION (POINT (5 1), LINESTRING (3 4, 4 4))')))", VARCHAR, "POLYGON ((3 1, 5 1, 5 4, 3 4, 3 1))");
     }
 
+    @Test
+    public void testSTEnvelopeAsPts()
+    {
+        assertFunction("transform(ST_EnvelopeAsPts(ST_GeometryFromText('MULTIPOINT (1 2, 2 4, 3 6, 4 8)')), x -> ST_AsText(x))", new ArrayType(VARCHAR), ImmutableList.of("POINT (1 2)", "POINT (4 8)"));
```

Comment:
To reduce copy-paste and improve readability, consider adding a helper method:

```
assertEnvelopeAsPts("MULTIPOINT (1 2, 2 4, 3 6, 4 8)", new Point(1, 2), new Point(4, 8));

private void assertEnvelopeAsPts(String wkt, Point lowerLeftCorner, Point upperRightCorner)
{
     assertFunction(format("transform(ST_EnvelopeAsPts(ST_GeometryFromText('%s')), x -> ST_AsText(x))", wkt), new ArrayType(VARCHAR), ImmutableList.of(new OGCPoint(lowerLeftCorner).asText(), new OGCPoint(upperRightCorner).asText()));
}
```

Please, add a test for empty geometry, e.g. `POLYGON EMPTY`.

Reply:
Done